support-lib/jni/djinni_support.cpp

```diff
@@ -371,6 +371,15 @@ jstring jniStringFromUTF8(JNIEnv * env, const std::string & str) {
     return res;
 }
 
+jstring jniStringFromWString(JNIEnv * env, const std::wstring & str) {
+
+    std::u16string utf16(str.begin(), str.end());;
```

Comment:
Is this portable? The C++ spec doesn't specify what character encoding `wstring` uses. There is no guarantee it's UTF-16.